cpp/ycm/CMakeLists.txt

```diff
@@ -103,16 +114,25 @@ if ( USE_CLANG_COMPLETER AND
     execute_process( COMMAND tar -xjf ${CLANG_FILENAME} )
   elseif( CLANG_FILENAME MATCHES ".+xz" )
     execute_process( COMMAND tar -xJf ${CLANG_FILENAME} )
+  elseif( CLANG_FILENAME MATCHES ".+exe" )
+    # NSIS tool is used to generate the Clang installer on Windows. Files
+    # from the installer can be extracted with 7-Zip.
+    execute_process( COMMAND 7z x ${CLANG_FILENAME} OUTPUT_QUIET )
```

Comment:
Yes, it is a requirement for building YCM with Clang support. I will document it in the `Windows Installation` section of the YCM `README.md`.